### PR TITLE
Fix close stale issues when user is commented.

### PR DESCRIPTION
**Pull request by shmishra99**
In this PR:
Add file: [stale-close-when-update.js](https://github.com/tensorflow/tensorflow/compare/master...shmishra99:tensorflow:master?expand=1#diff-39bf5eba4c1d80722ca595b737a62c1196331b5abaf9be01520eacda510fd8bf) close all issues and pull requests that have been marked as stale for more than 7 days for issues and 14 days for pull requests.
Update file: [.github/workflows/stale-issues.yml](https://github.com/tensorflow/tensorflow/compare/master...shmishra99:tensorflow:master?expand=1#diff-0bd376ac0f6d23a1457b2e910fd30cdd4cda10374ce15faa09c6ad26c3b82abe) to invoke the script from stale workflow.

### CODE REVIEW

**Comment by mihaimaruseac** [comment]
I think you only need to remove `remove-stale-when-updated: false`.

If the issue receives comments it should not be marked as stale, there are people that still care about it!

**Comment by shmishra99** [comment]
Thank you for your suggestion @mihaimaruseac. I have made the necessary changes to the workflow accordingly.

File: .github/workflows/stale-issues.yml [hunk 1]
```diff
@@ -77,4 +73,4 @@ jobs:
             It will be closed if no further activity occurs. Thank you.
           close-issue-message: >
             This issue was closed because it has been inactive for 1 year.
-          repo-token: ${{ secrets.GITHUB_TOKEN }}
+          repo-token: ${{ secrets.GITHUB_TOKEN }}
```
Comment (mihaimaruseac) [inline .github/workflows/stale-issues.yml:76]:
```suggestion
          repo-token: ${{ secrets.GITHUB_TOKEN }}

```

**Review by mihaimaruseac — COMMENTED** [review]
Adding a newline at the end of the file.

**Review by mihaimaruseac — APPROVED** [review]

**Comment by mihaimaruseac** [comment]
PS: Note that your git config is wrong, your email gets quoted in the commit (https://github.com/tensorflow/tensorflow/pull/60672/commits/6140088e0eb5fd9d759f237493a1bffb490d9026) and that required forcing the CLA to pass.

**Comment by shmishra99** [comment]
Hi @mihaimaruseac, 
I have modified my GitHub configuration. I also made a change so that if a user comments, it will also remove the stale dependent labels. This will eliminate the need for Probot to remove the labels.
Thank You!

**Review by ymodak — APPROVED** [review]

**Review by mihaimaruseac — APPROVED** [review]

**Pull request merged**